include/openssl/bn.h

```diff
@@ -136,14 +136,7 @@ int BN_is_word(const BIGNUM *a, const BN_ULONG w);
 int BN_is_odd(const BIGNUM *a);
 
 # define BN_one(a)       (BN_set_word((a),1))
-
-void BN_zero_ex(BIGNUM *a);
```

Comment:
This is very not ok. 

We do not remove functions from public header files like this or change their implementation without a very good reason. Your definition of "API" is somewhat unique in the team.